tests/letstest/testdata/sample-config/live/a.encryption-example.com/cert.pem

```diff
@@ -1 +0,0 @@
-../../archive/a.encryption-example.com/cert1.pem
```

Comment:
It looks like these files got changed from symlinks to files containing the path to the relevant file somehow.

The scary thing is, our tests still passed! It looks like our cleanup code in the integration tests is masking the real nonzero exit code.

Reply:
Sorry, I made the move on Windows: on this platform POSIX symlinks are represented as real files containing the path to the linked file. Fix that by repushing from Linux.

Yes, this is not very good. I already saw the tests failing, so I think in a way or another, it catches several potential errors, but clearly not all of them.

Soon certbot-ci will be there with its rewriting of these tests in Python. Then it will not be possible anymore to have some weird shell behavior hiding non zero exit codes.

Reply:
I checked the integration tests log directly, it is working now.